packages/cli-lib/templates.js

```diff
@@ -46,6 +50,7 @@ const isCodedFile = filePath => TEMPLATE_EXTENSION_REGEX.test(filePath);
 module.exports = {
   ANNOTATION_KEYS,
   getAnnotationValue,
+  getAnnotationsFromSource,
   buildAnnotationValueGetter,
```

Comment:
It looks like this actually might not be used anymore, if it makes more sense I can just change `buildAnnotationValueGetter` to use source instead of file

Reply:
> It looks like this actually might not be used anymore

It may not be used internally by us, but there's always the chance that it is being used externally (very unlikely, I know). I think what you currently have makes sense 👍